### DISCUSSION TITLE
Open item with id 0

### DISCUSSION THREAD

**Discussion by bentovsky** [Budibase/budibase#6031] [Help]
I have a table with several rows, and when I click on the "view" button, I'm being redirected to the item's details. But I'm having an issue when trying to open the item with the id 0.

![image](https://user-images.githubusercontent.com/16797919/169837818-d9e67d40-269c-43eb-904c-a3b98b5e4782.png)

**Comment by melohagan** [comment]
Hey @bentovsky 

What happens when you click on 'view'. Are there any errors?

What does the link URL look like? What does the route of your screen look like?

**Reply by bentovsky** [reply]
Thank you for the fast reply. I've realized that the issue was on my side. After investigating a bit, I've noticed that the ID linked with the "view" button was wrong.

**Reply by melohagan** [reply]
No problem, is that you sorted, or is there anything else I can help with?
